catalog/gcpm041u-lemur/projects_gcpm041u-lemur_assets_scanfi_v12_SCANFI_v1_2.jsonnet

```diff
@@ -0,0 +1,322 @@
+        // Only for bands with enumerated values.
+        'gee:classes': [
+          {value: 1, color: '000000', description: 'Artic Extrapolation Area'},
+        ],
+      },
+    ],
+
+    // Optional band statistics - one entry per band.
+    // If the exact statistics are known, then set gee:estimated_range to true.
+    // band_name_1: {minimum: 0, maximum: 255, 'gee:estimated_range': false},
+    // band_name_2: {minimum: 0, maximum: 1e8, 'gee:estimated_range': false},
+    // band_name_3: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    // band_name_4: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+
+    // One or more band visualizations.
+    'gee:visualizations': [
+      // Example with one band and a palette for the colors.
+      {
+        display_name: 'SCANFI Balsam Fir Percentage',
+        lookat: {lon: -94, lat: 56, zoom: 5},
+        image_visualization: {
+          band_vis: {
+            min: [0],
+            max: [100],
+            palette: ['ffffff', '05ff09'],
+            bands: ['balsamFir'],
+          },
+        },
+      },
+    ],
+  },
+  // The scientific extension.
+  // The best DOI that describes the *data*.
+  // Only use a research paper DOI if there is no dataset or data paper DOI.
+  // 'sci:doi': '10.23687/18e6a919-53fd-41ce-b4e2-44a9707c52dc',
+  // Use APA style for citations and publications. https://apastyle.apa.org/
+  'sci:citation': |||
+     Guindon L., Villemaire P., Correia D.L.P., Manka F., Lacarte S., Smiley B. 2023. 
+     SCANFI: Spatialized CAnadian National Forest Inventory data product. 
+     Natural Resources Canada, Canadian Forest Service, Laurentian Forestry Centre, Quebec, Canada. 
+     [https://doi.org/10.23687/18e6a919-53fd-41ce-b4e2-44a9707c52dc](https://doi.org/10.23687/18e6a919-53fd-41ce-b4e2-44a9707c52dc)
+     
+     Guindon, L., Manka, F., Correia, D.L.P., Villemaire, P., Smiley, B., Bernier, P., Gauthier, S., Beaudoin, A., Boucher, J., and Boulanger, Y. 2024. 
+     A new approach for Spatializing the Canadian National Forest Inventory (SCANFI) using Landsat dense time series. Can. J. For. Res. 
+     [https://doi.org/10.1139/cjfr-2023-0118](https://doi.org/10.1139/cjfr-2023-0118)
+  |||,
+  
+  // For standard SPDX licenses, use:
+  // 'gee:terms_of_use': ee.gee_terms_of_use(license),
+  // If there is a custom license (the license is set to spdx.proprietary, set
+  // gee:terms_of_use to enough text that a reader has a sense of what they are
+  // getting into. Be sure to add a link to the license in this Markdown text
+  // and add the link to the license section with ee.link.license(url).
+  'gee:terms_of_use': |||
+    Licensed under the
+    [Open Government Licence - Canada](https://open.canada.ca/en/open-government-licence-canada).
+  |||,
+  
+  // The fields below generally don't need to be changed.
+
+  // TODO(google): Remove gee:status when the dataset is ready.
+  'gee:status': 'incomplete',
```

Comment:
you can change the status from "incomplete" to "beta" to generate an unlisted dataset page. It will be publicly visible if you know the URL.